src/memory-analyzer/gdb_api.cpp

```diff
@@ -324,7 +333,7 @@ void gdb_apit::collect_malloc_calls()
     record = get_most_recent_record("*stopped");
   }
 
-  // now we can read the rax register to the the allocated memory address
+  // now we can read the rax register to the allocated memory address
```

Comment:
Since we're here, if you are editing again before merge then maybe `rax` should be generalised since `i386` will only have `eax`.

Reply:
Done!